src/main/java/com/google/devtools/build/lib/remote/RemoteOptions.java

```diff
@@ -46,6 +46,18 @@
   )
   public String remoteCacheProxy;
 
+    @Option(
+      name = "remote_s3_region",
```

Comment:
Use the remote_http_cache flag for this? The bucket should be able to have a unique URL?